src/rp2_common/pico_cyw43_driver/cybt_shared_bus/cybt_shared_bus_driver.c

```diff
@@ -559,10 +554,19 @@ cybt_result_t cybt_get_bt_buf_index(cybt_fw_membuf_index_t *p_buf_index) {
 static cybt_result_t cybt_reg_write(uint32_t reg_addr, uint32_t value) {
     cybt_debug("cybt_reg_write 0x%08lx 0x%08lx\n", reg_addr, value);
     cyw43_ll_write_backplane_reg(cyw43_ll, reg_addr, value);
+    if (reg_addr == HOST_CTRL_REG_ADDR)
+    {
+        host_ctrl_cache_reg = value;
+    }
```

Comment:
Maybe format it with the open curly brace on the same line as the `if`, to match surrounding code?